Den haag feature/add app colors

ThoriaHamdy99 opened the pull request:
Den-Haag Add app colors

ThoriaHamdy99 committed:
define cute tudee app colors.

ThoriaHamdy99 committed:
add dark theme colors.

marah222 reviewed (changes requested):
Great Job here, I suggest to follow the Git hook in naming commits

Noor-Serry commented:
Great. I have one comment only what about moving the color , dark theme color and light theme color file to one package inside the theme package

monaayman123 reviewed (commented):
i suggest to start a commit with uppercase letter

ThoriaHamdy99 committed:
move the app colors to colors package under theme package.

ThoriaHamdy99 commented:
> Great Job here, I suggest to follow the Git hook in naming commits

I didn't understand you. is there is something wrong on the commits?

ThoriaHamdy99 commented:
> Great. I have one comment only what about moving the color , dark theme color and light theme color file to one package inside the theme package

Yes you are right just I forgot to add them to a sub package.  I added colors package and moved everything related to colors to it . Please check and approve if there is no another feedback.

ThoriaHamdy99 commented:
> i suggest to start a commit with uppercase letter

I don't agree for that. the commits should start with lowercase
